master/internal/resourcemanagers/fair_share.go

```diff
@@ -51,6 +51,12 @@ func (f *fairShare) Schedule(rp *ResourcePool) ([]*sproto.AllocateRequest, []*ac
 	return fairshareSchedule(rp.taskList, rp.groups, rp.agents, rp.fittingMethod)
 }
 
+func (f *fairShare) OrderedAllocations(
+	rp *ResourcePool,
+) (reqs []*sproto.AllocateRequest) {
+	return reqs
```

Comment:
This is WIP right? Else I'm flagging it as incorrect :)

Reply:
wdy mean, are you saying I couldn't have solved this in one line of code? : ) 
joking aside it's just like this because of scope (out of scope)